includes/classes/wp-maintenance-mode.php

```diff
@@ -1067,6 +1067,18 @@ public function add_maintenance_template( $templates ) {
 		 */
 		public function use_maintenance_template( $template ) {
 			global $post;
+
+			// Return the default template if Elementor edit mode.
+			if ( class_exists( '\Elementor\Plugin', false ) && ( isset( $_GET['action'] ) && 'elementor' === $_GET['action'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+				return $template;
+			}
+
+			// Return the default template if Elementor preview mode.
+			if ( class_exists( '\Elementor\Plugin', false ) && isset( $_GET['elementor-preview'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+				return $template;
+			}
```

Comment:
Looking nice. One small preference, we can group the conditions like this along with the explanations:

```php
// Return the default template for Elementor when:
if (
    class_exists( '\Elementor\Plugin', false ) &&
    (
        // Edit Mode is on.
        ( isset( $_GET['action'] ) && 'elementor' === $_GET['action'] ) || // phpcs:ignore WordPress.Security.NonceVerification.Recommended
        // Preview Mode is on.
        isset( $_GET['elementor-preview'] ) // phpcs:ignore WordPress.Security.NonceVerification.Recommended
    )
) { 
    return $template;
}
```

Reply:
Done! Thank you for review